### PR TITLE
Add lib3ds 2.0.0

### PR DESCRIPTION
**Pull request by AdrianArroyoCalle**
http://code.google.com/p/lib3ds but since Google Code is going to close I use my own GitHub backup (http://github.com/AdrianArroyoCalle/lib3ds)

### CODE REVIEW

File: media-libs/lib3ds/lib3ds-2.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:cmake
+	"
+
+BUILD()
+{
+	cmake .
```
Comment (waddlesplash) [inline media-libs/lib3ds/lib3ds-2.0.0.recipe]: Add `$cmakeDirArgs` here.

**Comment by waddlesplash** [comment]
Please squash your commits.

**Comment by AdrianArroyoCalle** [comment]
Done!

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**